src/SDK/SmartDetectorsPackage/SmartDetectorsPackage.nuproj

```diff
@@ -22,7 +22,7 @@
   <Import Project="$(NuProjPath)\NuProj.props" Condition="Exists('$(NuProjPath)\NuProj.props')" />
   <PropertyGroup Label="Configuration">
     <Id>Microsoft.Azure.Monitoring.SmartDetectors</Id>
-    <Version>0.9.6.0</Version>
+    <Version>0.10.0.0</Version>
```

Comment:
>0.10.0.0 [](start = 13, length = 8)

0.9.7.0

Reply:
but why did you change the version?

---
In reply to: [230674652](https://github.com/Azure/SmartDetectorsMonitoringAppliance/pull/16#discussion_r230674652) [](ancestors = 230674652)

Reply:
oh got it - but still 9.7

---
In reply to: [230674758](https://github.com/Azure/SmartDetectorsMonitoringAppliance/pull/16#discussion_r230674758) [](ancestors = 230674758,230674652)